src/api/protectedApiClient.ts

```diff
@@ -107,13 +132,35 @@ const getRequestStatuses = (): Promise<PersonalRequest[]> => {
 };
 
 const makePFRequest = (): Promise<void> => {
-  return AppAxiosInstance.post(ProtectedApiClientRoutes.MAKE_PF_REQUEST)
+  return AppAxiosInstance.post(ProtectedApiClientRoutes.PF_REQUESTS)
     .then((res) => {
       return;
     })
     .catch((err) => err);
 };
 
+const getPFRequests = (): Promise<PFRequestResponse> => {
+  return AppAxiosInstance.get(ProtectedApiClientRoutes.PF_REQUESTS);
```

Comment:
Here we dont want to return the whole axios response, we just want the data
```suggestion
  return AppAxiosInstance.get(ProtectedApiClientRoutes.PF_REQUESTS).then(res => res.data);
```